src/devcenter/azext_devcenter/vendored_sdks/devcenter_dataplane/operations/_operations.py

```diff
@@ -1737,12 +1737,12 @@ def build_deployment_environments_create_or_update_environment_request(
 
 
 def build_deployment_environments_delete_environment_request(
-    project_name: str, environment_name: str, user_id: str = "me", **kwargs: Any
+    project_name: str, environment_name: str, user_id: str = "me", force: Optional[bool] = None, **kwargs: Any
 ) -> HttpRequest:
     _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
     _params = case_insensitive_dict(kwargs.pop("params", {}) or {})
 
-    api_version: str = kwargs.pop("api_version", _params.pop("api-version", "2024-05-01-preview"))
+    api_version: str = kwargs.pop("api_version", _params.pop("api-version", "2024-10-01-preview"))
```

Comment:
Here

Reply:
What is happening here is that if an api-version is not provided, it will use 2024-10-01-preview. However, it seems the default is being passed in.